webpack.config.js

```diff
@@ -1,6 +1,7 @@
 const HtmlWebPackPlugin = require('html-webpack-plugin');
 
 module.exports = {
+  entry: ['@babel/polyfill', './src/index.jsx'],
```

Comment:
Why do you need this polyfill?

Reply:
Without ```@babel/polyfill```, a "regeneratorRuntime is not defined" error occurs when trying to use async/await.